libs/platform/src/lib/components/form/radio/radio.component.ts

```diff
@@ -0,0 +1,175 @@
+import {
+    Component,
+    ChangeDetectorRef,
+    EventEmitter,
+    Input,
+    Output,
+    Optional,
+    Self,
+    ViewChild,
+    ElementRef,
+    ChangeDetectionStrategy
+} from '@angular/core';
+import { RadioButtonComponent, stateType } from '@fundamental-ngx/core';
+import { ControlValueAccessor, NgControl } from '@angular/forms';
+import { FormFieldControl, InputSize } from '../form-control';
+
+@Component({
+    selector: 'fdp-radio-button',
+    templateUrl: './radio.component.html',
+    styleUrls: ['./radio.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [{ provide: FormFieldControl, useExisting: PlatformRadioButtonComponent, multi: true }]
+})
+export class PlatformRadioButtonComponent implements ControlValueAccessor {
+    /** name of radio button, will be set from radio button group name */
+    @Input()
+    get name(): any {
```

Comment:
I am finishing my review as the same applies for name, its not addressed, Please come back when you address all, or at least you raise question.

I don't feel like copying over all what I said... Please next time, dont ask for input on slack..

Reply:
i had not pushed changes for Radio button group. Now i have pushed.
yes slack is not a good place for comments.